lib/etcdv3.rb

```diff
@@ -45,6 +45,18 @@ def token
     request.token
   end
 
+  def key
+    File.read(File.expand_path(@options.fetch(:key)))
+  end
+
+  def cert
+    File.read(File.expand_path(@options.fetch(:cert)))
+  end
+
+  def cacert
+    File.read(File.expand_path(@options.fetch(:cacert)))
+  end
+
```

Comment:
I think we may want to return `nil` in the event someone hits this method when the option isn't present.  
```
irb(main):017:0* conn.key
KeyError: key not found: :key
	from /Users/shaun/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/etcdv3-0.6.0/lib/etcdv3.rb:49:in `fetch'
	from /Users/shaun/.rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/etcdv3-0.6.0/lib/etcdv3.rb:49:in `key'
	from (irb):17
	from /Users/shaun/.rbenv/versions/2.2.4/bin/irb:11:in `<main>'
```

Maybe something like this? 
```ruby
File.read(File.expand_path(@options[:cacert])) if @options[:cacert]
```